src/main/java/core/basesyntax/Printer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Printer {
+    public void drawFigure();
```

Comment:
```suggestion
    public void draw();
```

Reply:
1) removed unnecessary methods and seters from the class Figure
2) Added ENUM "Figures" and implement it in ColorSupplier
3) Rename Interface "Print" to "Drawable" and his method "DrawFigure" to "Draw"
4) All properities of getDefaultFigure() figure in "FigureSuppllier" is replaced by constants